### ISSUE TITLE
NYDailyNews (partial) breaks images (fonts?)

### ISSUE THREAD

**Issue by terrorist96** [EFForg/https-everywhere#11194]
http://www.nydailynews.com/entertainment/music/mobb-deep-rapper-prodigy-dead-42-article-1.3262918

`NYDailyNews (partial)` breaks some images (fonts?).

![image](https://user-images.githubusercontent.com/12802056/28289654-70366ad4-6b11-11e7-93d5-a9199e51a635.png)

**Comment by koops76** [comment]
Can you post a developer tools log?

**Comment by terrorist96** [comment]
Could you please give me a step-by-step on how to do that?

**Comment by koops76** [comment]
F12 → Console → right click anywhere → Save as...

**Comment by koops76** [comment]
Also WTF @ your nickname.

**Comment by terrorist96** [comment]
[www.nydailynews.com-1500335216954.zip](https://github.com/EFForg/https-everywhere/files/1154287/www.nydailynews.com-1500335216954.zip)

**Comment by koops76** [comment]
Post it as text without archiving. If it's large post it on Gist.

**Comment by terrorist96** [comment]
Don't trust me? :stuck_out_tongue_closed_eyes: 
https://gist.github.com/terrorist96/c1c9710e3c3a5b855d7e0402e7bfbe15

**Comment by koops76** [comment]
Your log doesn't show any issue. I tried visiting this page with the ruleset both on and off. No difference. Try disabling all extensions except HTTPS Everywhere and see if images are still broken.

**Comment by koops76** [comment]
I have uBlock Origin and Privacy Badger but neither block any images on this page.

**Comment by terrorist96** [comment]
I have those too. But here is a screenshot from a new profile with only HTTPS Everywhere:
![image](https://user-images.githubusercontent.com/12802056/28330441-97996f90-6bbb-11e7-9170-f9286792af34.png)

And here is an image on the same profile with HTTPS Everywhere active but the NYDailyNews rule deactivated:
![image](https://user-images.githubusercontent.com/12802056/28330495-ca94d97a-6bbb-11e7-970c-a68d3c37d044.png)

**Comment by numismatika** [comment]
There is a lot of mixed content warnings scrolling by:

> http://platform.twitter.com/widgets.j
> http://asset.pagefair.com/measure.min.js
> http://assets.nydailynews.com/nydn/js/jquery-ui.js?r=201706290002
> http://c2.taboola.com/nr/nydailynews-nydailynews/newsroom.js
> http://pixel.quantserve.com/aquant.js?a=p-af_gBGFZgopbs
> http://s.ntv.io/serve/load.js

the important one probably nydailynews.com

**Comment by koops76** [comment]
@terrorist96 Those are fonts, not images.

**Comment by koops76** [comment]
```
mobb-deep-rapper-prodigy-dead-42-article-1.3262918:1 Access to Font at 'https://www.nydailynews.com/nydn/f/themify.woff?-fvbane' from origin 'http://www.nydailynews.com' has been blocked by CORS policy: The 'Access-Control-Allow-Origin' header has a value 'http://interactiveqa.nydailynews.com' that is not equal to the supplied origin. Origin 'http://www.nydailynews.com' is therefore not allowed access.
mobb-deep-rapper-prodigy-dead-42-article-1.3262918:1 Access to Font at 'https://www.nydailynews.com/nydn/f/themify.ttf?-fvbane' from origin 'http://www.nydailynews.com' has been blocked by CORS policy: The 'Access-Control-Allow-Origin' header has a value 'http://interactiveqa.nydailynews.com' that is not equal to the supplied origin. Origin 'http://www.nydailynews.com' is therefore not allowed access.
```

**Comment by koops76** [comment]
@terrorist96 #11229

**Comment by jeremyn** [comment]
This is fixed by PR #11514 and works by not rewriting `assets.nydailynews.com` to `www.nydailynews.com`. The font file(s) from `http://assets` have sufficient CORS permissions (the `Access-Control-Allow-Origin` header) but the font file(s) from `https://www` don't.
